src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+  function rt(t) {
+    var e = t.ownerDocument.defaultView.getComputedStyle(t),
+      n = parseFloat(e.marginTop || 0) + parseFloat(e.marginBottom || 0),
+      i = parseFloat(e.marginLeft || 0) + parseFloat(e.marginRight || 0);
+    return { width: t.offsetWidth + i, height: t.offsetHeight + n };
+  }
+  function st(t) {
+    var e = { left: "right", right: "left", bottom: "top", top: "bottom" };
+    return t.replace(/left|right|bottom|top/g, function (t) {
+      return e[t];
+    });
+  }
+  function at(t, e, n) {
+    n = n.split("-")[0];
+    var i = rt(t),
+      o = { width: i.width, height: i.height },
+      r = -1 !== ["right", "left"].indexOf(n),
+      s = r ? "top" : "left",
+      a = r ? "left" : "top",
+      l = r ? "height" : "width",
+      c = r ? "width" : "height";
+    return (
+      (o[s] = e[s] + e[l] / 2 - i[l] / 2),
+      (o[a] = n === a ? e[a] - i[c] : e[st(a)]),
+      o
+    );
+  }
+  function lt(t, e) {
+    return Array.prototype.find ? t.find(e) : t.filter(e)[0];
+  }
+  function ct(t, e, n) {
+    return (
+      (void 0 === n
+        ? t
+        : t.slice(
+            0,
+            (function (t, e, n) {
+              if (Array.prototype.findIndex)
+                return t.findIndex(function (t) {
+                  return t[e] === n;
+                });
+              var i = lt(t, function (t) {
+                return t[e] === n;
+              });
+              return t.indexOf(i);
+            })(t, "name", n)
+          )
+      ).forEach(function (t) {
+        t.function &&
+          console.warn("`modifier.function` is deprecated, use `modifier.fn`!");
+        var n = t.function || t.fn;
+        t.enabled &&
+          O(n) &&
+          ((e.offsets.popper = K(e.offsets.popper)),
+          (e.offsets.reference = K(e.offsets.reference)),
+          (e = n(e, t)));
+      }),
+      e
+    );
+  }
+  function ut() {
+    if (!this.state.isDestroyed) {
+      var t = {
+        instance: this,
+        styles: {},
+        arrowStyles: {},
+        attributes: {},
+        flipped: !1,
+        offsets: {},
+      };
+      (t.offsets.reference = ot(
+        this.state,
+        this.popper,
+        this.reference,
+        this.options.positionFixed
+      )),
+        (t.placement = it(
+          this.options.placement,
+          t.offsets.reference,
+          this.popper,
+          this.reference,
+          this.options.modifiers.flip.boundariesElement,
+          this.options.modifiers.flip.padding
+        )),
+        (t.originalPlacement = t.placement),
+        (t.positionFixed = this.options.positionFixed),
+        (t.offsets.popper = at(this.popper, t.offsets.reference, t.placement)),
+        (t.offsets.popper.position = this.options.positionFixed
+          ? "fixed"
+          : "absolute"),
+        (t = ct(this.modifiers, t)),
+        this.state.isCreated
+          ? this.options.onUpdate(t)
+          : ((this.state.isCreated = !0), this.options.onCreate(t));
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)